model/error.go

```diff
@@ -0,0 +1,23 @@
+package model
+
+import (
+	"time"
+
+	"google.golang.org/appengine/datastore"
+)
+
+//go:generate generator
+
+// Error contains the data of an
+// error encountered by the user.
+type Error struct {
+	time.Time
+	Status int
+	User   *datastore.Key
+	// The description of the events that
```

Comment:
Please add a newline before comments like this, so that it is visually easier to distinguish which property the comment means.